src/screens/UserManagement/UserRevamp/InviteMember.res

```diff
@@ -0,0 +1,151 @@
+@react.component
+let make = (~isInviteUserFlow=true, ~setNewRoleSelected=_ => ()) => {
+  open APIUtils
+  open LogicUtils
+
+  let getURL = useGetURL()
+  let updateDetails = useUpdateMethod()
+  let showToast = ToastState.useShowToast()
+  let {email} = HyperswitchAtom.featureFlagAtom->Recoil.useRecoilValueFromAtom
+  let (loaderForInviteUsers, setLoaderForInviteUsers) = React.useState(_ => false)
+  let authId = HyperSwitchEntryUtils.getSessionData(~key="auth_id")
```

Comment:
is authId is used

Reply:
Its passed in `Invite_Multiple` api . For SSO case